jscomp/others/js_bigint.ml

```diff
@@ -0,0 +1,31 @@
+(** JavaScript BigInt API *)
+
+type t
+
+external ofString : string -> t = "BigInt" [@@bs.val]
+external ofInt : int -> t = "BigInt" [@@bs.val]
+external ofFloat : float -> t = "BigInt" [@@bs.val]
```

Comment:
This is dangerous enough that I think we may not want to include it.

The ReScript expression `ofFloat(99999999999999999999999999999.)` for example will produce the value `99999999999999991433150857216n` as the float literal will be truncated by the JS environment.

Thoughts?

Reply:
What OCaml version is being used to compile ReScript nowadays? Newer OCaml versions have an 'alert' feature that trigger a warning if you use some piece of code. E.g. I'm doing that in https://github.com/yawaramin/ocaml-decimal/blob/08c57183c8673b5058bd6010570e69f0201c03c7/lib/decimal.mli#L321

If that's not available, perhaps we can mark this function as deprecated with the `@deprecated` tag.

Reply:
`alert` doesn't seem to be available: https://github.com/rescript-lang/rescript-compiler/blob/master/jscomp/ml/builtin_attributes.mli